### ISSUE TITLE
Own server deprecated?

### ISSUE THREAD

**Issue by Myth0ne** [Adsmurai-Google-Tag-Manager-Templates/adsmurai-facebook-pixel-and-conversions-api#23]
Why has the template been modified 
'I have my own server (deprecated)'

Are we to no longer use this? 

What is the best way to implement FB Pixel & Conversion API tracking on a custom server side tracking container?

**Comment by mjadsmurai** [comment]
Because the selfhosted template only supports FB while our serveerless solution has been growing to support more providers like Tiktok, Pinterest, Linkedin, etc.. Some of our clients were confused to not have those functionalities when selfhosting, so we updated the text to make it clear that selhosted is an old option.

**Comment by Myth0ne** [comment]
Ok it was just a bit confusing as I wasn't sure if it was non-functional or not. Maybe that should be clarified that it will still work but only for FB.

Is there a guide on the Adsmurai Serverless solution to setup FB pixel and Google Tag manager etc?

**Comment by mjadsmurai** [comment]
> Ok it was just a bit confusing as I wasn't sure if it was non-functional or not. Maybe that should be clarified that it will still work but only for FB.

The thing is that it may stop working and we won't be fixing it when that happens.

> Is there a guide on the Adsmurai Serverless solution to setup FB pixel and Google Tag manager etc?

Yes! on https://help.adsmurai.com/one-tag#providers you can see the tutorials for each provider.

**Comment by Myth0ne** [comment]
Do you accept google pay or paypal methods? Waiting on a new card to arrive and cannot set up the one tag. Looks like card only?

**Comment by mjadsmurai** [comment]
Since payments are processed using Stripe i'm afraid we don't have the option for paypal :( . But i will ask the team if Google Pay can be enabled.

**Comment by Myth0ne** [comment]
Any luck at all?

**Comment by Myth0ne** [comment]
One last question, does Adsmurai One Tag only support browser side events? 
Is there a way it achieves both browser and server side tagging to deduplicate events effortlessly? How is this setup? Your video is super brief and only shows adding the page view tag and adding the API token from the One Tag dashboard

**Comment by Myth0ne** [comment]
Ok thanks, so in this setup, would you simply just run 1 Google Tag web container and remove the server container and use this method instead to achieve deduplication events?

**Comment by Myth0ne** [comment]
How do you also achieve doing test events in FB?

E.g. setting TEST12345 to test page view and other related events?

**Comment by mjadsmurai** [comment]
> Ok thanks, so in this setup, would you simply just run 1 Google Tag web container and remove the server container and use this method instead to achieve deduplication events?

Exactly, with OneTag you just have to use https://tagmanager.google.com/gallery/#/owners/Adsmurai-Google-Tag-Manager-Templates/templates/adsmurai-facebook-pixel-and-conversions-api
as the server side is handled by us.

> How do you also achieve doing test events in FB?
> 
> E.g. setting TEST12345 to test page view and other related events?

This is something you can find on the Server event parameters section of the tag:
![Screenshot from 2024-09-20 14-13-33](https://github.com/user-attachments/assets/b10b86d8-46df-4e9a-babb-d08c6f68d6a4)

**Comment by Myth0ne** [comment]
Ok great thanks for your help.

**Comment by Myth0ne** [comment]
One last question, 
By default do you recommend to enable "Hash user data on the server" option? I don't believe anything is hashed but is this recommended or just to leave off by default?

**Comment by mjadsmurai** [comment]
If you don't set the hashed data on the tag then yes, enabled this option.
Otherwise you will see in the Health dashboard that some providers will tell you the data is not hashed. (some try to hash it themselves)

**Comment by Myth0ne** [comment]
> If you don't set the hashed data on the tag then yes, enabled this option.

how do we confirm if we dont see hashed data on the tag?

**Comment by Myth0ne** [comment]
Oh sorry, I think I misread this. Then yes I will enable this.
